mmpretrain/configs/swin_transformer_v2/swinv2_large_w12_8xb128_in21k_192px.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# This is a BETA new format config file, and the usage may change recently.
+from mmengine.config import read_base
+
+with read_base():
+    from .._base_.datasets.imagenet21k_bs128 import *
+    from .._base_.default_runtime import *
+    from .._base_.models.swin_transformer_v2_base import *
+    from .._base_.schedules.imagenet_bs1024_adamw_swin import *
+
+# model settings
+model = dict(
+    backbone=dict(img_size=192, window_size=[12, 12, 12, 6]),
```

Comment:
set arch, please check other configs

Reply:
> set arch, please check other configs

The content here is consistent with [configs/swin_transformer_v2/swinv2-base-w12_8xb128_in21k-192px.py](https://github.com/open-mmlab/mmpretrain/blob/main/configs/swin_transformer_v2/swinv2-base-w12_8xb128_in21k-192px.py) before transplantation. In the original file, although `base_256.py` was introduced, `model` was redefined in the new file. In fact, the `model` content of `base_256.py` has been covered. I am wondering whether it is necessary to set arch here? @fangyixiao18